src/Sylius/Bundle/CoreBundle/Settings/TaxationSettingsSchema.php

```diff
@@ -26,15 +28,13 @@
 class TaxationSettingsSchema implements SchemaInterface
 {
     /**
-     * Zone repository.
+     * Zone repository
      *
      * @var ObjectRepository
      */
-    private $zoneRepository;
+    protected $zoneRepository;
```

Comment:
I'll change it back. I was just trying to be consistent with the scope of their other member variables.